src/favicon/favicon.py

```diff
@@ -40,12 +40,14 @@
 Icon = namedtuple('Icon', ['url', 'width', 'height', 'format'])
 
 
-def get(url, *args, **request_kwargs):
+def get(url, *args, html=None, **request_kwargs):
     """Get all fav icons for a url.
 
     :param url: Homepage.
     :type url: str
 
+    :param html: HTML input, as string. Will be used instead of an HTTP response from the `url`.
+
```

Comment:
1. Can you rename `html` to `html_override` so it is clear why a user would pass a value to it?
2. Add `:type html: str or None`

Reply:
Done